test/com/dotcms/util/marshal/MarshalUtilsTest.java

```diff
@@ -35,6 +38,12 @@
  */
 
 public class MarshalUtilsTest {
+	
+	@BeforeClass
+	public static void prepare() throws Exception{
+		//Setting web app environment
+        IntegrationTestInitService.getInstance().init();
```

Comment:
If the IntegrationTestInitService isn't called one test fails: marshalSystemEvent